apis/token-price/api/v2/[chainId]/[address].ts

```diff
@@ -10,14 +10,20 @@ const schema = z.object({
     .int()
```

Comment:
Can add a check for isExtractorSupportedChainId in here

Reply:
Yup, was lazy and used v1 schema